go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/go-sourcemap/sourcemap v2.1.4-0.20211119122758-180fcef48034+incompatible
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/websocket v1.5.0
+	github.com/grafana/xk6-browser v0.7.0
```

Comment:
Run `go mod tidy` and `go mod vendor` and the CI issues should disappear

Reply:
Resolved in 85becc294d1975e90f8c3a480fe1155b59c916c1